connaisseur/tests/integration/integration-test.sh

```diff
@@ -4,9 +4,9 @@ set -euo pipefail
 # This script is expected to be called from the root folder of Connaisseur
 
 echo 'Preparing Connaisseur config...'
-export RELEASE_IMAGE=$(yq r helm/values.yaml 'deployment.image')
+export RELEASE_IMAGE=$(yq e '.deployment.image' helm/values.yaml)
 envsubst < connaisseur/tests/integration/update-config.yaml > update
-yq write --inplace --script update helm/values.yaml # Coincidentally, this also ensure all used env variables are set
```

Comment:
@Starkteetje This breaks the coincidental checking for unset env vars in `update-config.yaml`. Do you have any idea how to reintroduce it here?

Reply:
```yq eval-all --null-input ".a=env(myenv)"
Error: Value for env variable 'myenv' not provided in env()```
seems like it will reject them if defined with `env`.Might work with files as well 🤷

Reply:
no, I tested. However, what are we trying to achieve here actually? Am I reading it correctly, that we cut the deployment image from the `helm/values.yml` to update the `helm/values.yml` with its value? What am I missing?

Reply:
We also update the root key and ensure some other settings as described by https://github.com/sse-secure-systems/connaisseur/blob/master/connaisseur/tests/integration/update-config.yaml
But yes, the env variable depended stuff has apparently become obsolete